.github/workflows/translations-sync.yml

```diff
@@ -3,27 +3,27 @@
 
 # See translations-upload.yml for automation to upload our source content
 # See translations-pr-lint.yml for quality control we conduct on ingress of new translations.
-name: Crowdin Download
+name: Crowdin Translations Sync
 
 on:
-  workflow_dispatch: # Allow running when we want to, for events such as urgent translation mistakes or 100% completed languages
+  workflow_dispatch: # Manual trigger for urgent fixes
   schedule:
-    - cron: '0 5 * * 5' # At 05:00 on Fridays. This guarantees that we have the 72 hour weekend time to review translations.
```

Comment:
I feel like we are losing something here - perhaps we should link explicitly to the 72 hour policy, here and maybe in the PR. We have had a number of problems with that.